addOns/paramminer/src/main/java/org/zaproxy/addon/paramminer/ParamMinerConfig.java

```diff
@@ -321,4 +326,36 @@ public void setUrlGetRequest(boolean urlGetRequest) {
     public boolean getUrlGetRequest() {
         return urlGetRequest;
     }
+
+    public void setCustomHeaderWordlistPath(String text) {
```

Comment:
The config could have the lists of files, it would not require so many properties and it would simplify its usage.

Reply:
This can be handled in the sets of optimization PRs :). The MVP should be ready first .. Alpha 0 release .. Alpha 1 release and then before we move onto beta :). I can make a PRs optimizing the overall extension :)

Reply:
Sounds good.